src/Raven.Client/Json/BlittableOperation.cs

```diff
@@ -56,6 +56,11 @@ public static bool EntityChanged(BlittableJsonReaderObject newObj, DocumentInfo
             {
                 if (changes == null)
                     return true;
+                if (field.Equals(LastModified) ||
+                    field.Equals(Collection) ||
```

Comment:
Why are we ignoring the collection field or the id here?

Reply:
We already do that 10 lines below for new fields.

Reply:
Ccollection doesn't need to be there but we are removing the id in JsonConverter.ToBlittable and we don't want to mark that as removed

Reply:
ok